<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>481</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="main_layout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="Predect_label">
         <property name="text">
          <string>Predicted:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="predicted_value">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="confidence_label">
         <property name="text">
          <string>Confidence:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="confidence_percentage">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="image_holder">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="confidence_label_2">
         <property name="text">
          <string>Image path</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="image_path"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="load_image_button">
       <property name="text">
        <string>Load Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="predict_button">
       <property name="text">
        <string>Predict</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
